Singer.API/ClientApp/src/app/modules/dashboard/components/home/home.component.ts

```diff
@@ -1,42 +1,24 @@
 import { Component, OnInit } from '@angular/core';
-import { EventDescription } from 'src/app/modules/core/models/singerevent.model';
+import {
+   EventDescription,
+   SearchEventDTO,
+} from 'src/app/modules/core/models/singerevent.model';
+import { Observable } from 'rxjs';
+import { PaginationDTO } from 'src/app/modules/core/models/pagination.model';
+import { HttpParams } from '@angular/common/http';
+import { map } from 'rxjs/operators';
+import { ApiService } from 'src/app/modules/core/services/api.service';
 import { SearchEventData } from '../event-search/event-search.component';
 import { SingerEventLocation } from 'src/app/modules/core/models/singer-event-location';
 import { AuthService } from 'src/app/modules/core/services/auth.service';
-import { SingerEventsService } from 'src/app/modules/core/services/singerevents-api/singerevents.service';
-import { SingerEventLocationService } from 'src/app/modules/core/services/singerevents-api/singerevent-location.service';
 
 @Component({
    selector: 'app-home',
    templateUrl: './home.component.html',
    styleUrls: ['./home.component.css'],
 })
 export class HomeComponent implements OnInit {
```

Comment:
If the ngOnInit() is empty, you could remove the OnInit implementation too.

Reply:
Done.